virtcontainers/container.go

```diff
@@ -524,6 +533,7 @@ func (c *Container) mountSharedDirMounts(hostSharedDir, guestSharedDir string) (
 			if err := c.sandbox.devManager.AttachDevice(m.BlockDeviceID, c.sandbox); err != nil {
```

Comment:
`err` is a new variable and the previous `err` that you declared is not considered.  For example: 

```go
package main

import "fmt"

func f() error {
	return fmt.Errorf("foo")
}

func main() {
	var err error
	if err := f(); err != nil {
		fmt.Printf("#1: %v\n", err)
	}
	fmt.Printf("#2 error: %v\n", err)
}
```

**output**
```
#1 foo
#2 <nil>
```